mycli/myclirc

```diff
@@ -65,6 +65,9 @@ less_chatty = False
 # Use alias from --login-path instead of host name in prompt
 login_path_as_host = False
 
+# Enable auto-verical-output
```

Comment:
A little bit more description in terms of what this actually does would be nice. Or a link to the documentation for this option. Such as https://dev.mysql.com/doc/refman/5.5/en/mysql-command-options.html#option_mysql_auto-vertical-output